CHANGELOG.md

```diff
@@ -128,6 +128,13 @@ The `.editorconfig` property `disabled_rules` (api property `DefaultEditorConfig
 
 Although, Ktlint 0.47.0 falls back on property `disabled_rules` whenever `ktlint_disabled_rules` is not found, this result in a warning message being printed. 
 
+#### Default/alternative .editorconfig
+
+Parameter "ExperimentalParams.editorConfigDefaults" is deprecated in favor of the new parameter "ExperimentalParams.editorConfigDefaults". When used in the old implementation this resulted in ignoring all ".editorconfig" files on the path to the file. The new implementation uses properties from the "editorConfigDefaults"parameter only when no ".editorconfig" files on the path to the file supplies this property for the filepath.
```

Comment:
Is this a typo? Both params are named `ExperimentalParams.editorConfigDefaults`

Reply:
Yes indeed a typo. Fixed it.